lib/adal-angular.js

```diff
@@ -320,7 +320,7 @@ if (typeof module !== 'undefined' && module.exports) {
                             if (authService.loginInProgress()) {
                                 authService.info('login already start.');
                                 return $q.reject();
-                            } else if (authService.config && isEndpoint) {
+                            } else if (authService.config && (!authService.config.endpoints || isEndpoint)) {
```

Comment:
@tushargupta51 I agree to your point.  The check on line 311-317 is duplicate.  But I have a question.. if line#297 gives a non-null 'resource', how the same duplicate check failed to update the flag isEndpoint as true?  Does this mean, the check on #298 is wrong too ?